controllers/Index.php

```diff
@@ -228,4 +228,9 @@ private function loadItems($options, $tags) {
             'hasMore' => $itemDao->hasMore()
         ];
     }
+
+    public function appCacheManifest() {
+        $this->view->genAppcacheManifest();
+        $this->view->sendfile('public/selfoss.appcache');
```

Comment:
Either way, I would let the web server serve the file like we do with CSS.

Reply:
Yes I did that at first. But is did not behave well with updates css/js updates.